__tests__/run_seed.test.js

```diff
@@ -154,12 +156,53 @@ describe("GET /article/comments", () => {
         });
     });
     test("Returns a 400 error message for invalid id", () => {
-        return request(app)
-          .get("/api/articles/hello")
-          .expect(400)
-          .then(({ body: { msg } }) => {
-            expect(msg).toBe("Bad request!");
-          });
-      });
+      return request(app)
+        .get("/api/articles/hello")
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Bad request!");
+        });
+    });
+  });
+});
+describe("POST", () => {
+  describe("POST responds request body accepts a comment object and responds with the post comment", () => {
+    test("Returns a status 201 and responds with the updated comment object", () => {
+      const updatedComment = { username: "lurker", body: "blah blah blah" };
+      const responseComment = {
+        comment: {
+          article_id: 3,
+          author: "lurker",
+          body: "blah blah blah",
+          comment_id: 19,
+          created_at: expect.any(String),
+          votes: 0,
+        },
+      }
+      return request(app)
+        .post(`/api/articles/3/comments`)
+        .send(updatedComment)
+        .expect(201)
+        .then((response) => {
+          const { body } = response;
+          expect(body).toEqual(responseComment);
+        });
+    });
+    test("Returns a 404 error message for article id not found", () => {
+      return request(app)
+        .post("/api/articles/2345678/comments")
+        .expect(404)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Not found!");
+        });
+    });
+    test("Returns a 400 error message for invalid id", () => {
+      return request(app)
+        .post("/api/articles/hello/comments")
+        .expect(400)
+        .then(({ body: { msg } }) => {
+          expect(msg).toBe("Bad request!");
+        });
+    });
   });
```

Comment:
Another test might be that they use a username that doesn't exist within the test database ( 404)